.github/workflows/phpunit.yml

```diff
@@ -31,5 +32,5 @@ jobs:
         run: |
           echo "::add-matcher::${{ runner.tool_cache }}/php.json"
           echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
-      - run: composer install
+      - run: composer install --ignore-platform-req=php+
```

Comment:
Hey all sorry for not paying a lot of attention over here. Now that #99 is merged I suppose this MR is obsolete?

Do we actually need this line change here? Seems to be it works fine without but I also didn't touch PHP in years :sweat_smile:

Reply:
This change was necessary to allow the installation of some dependency that was not yet marked as compatible with PHP 8.2 for CI testing purposes. It basically says that any constraints excluding higher PHP versions are ignored. Now that PHP 8.2 is released, this line is likely no longer necessary, because everything is "officially" compatible.